Makefile

```diff
@@ -433,7 +433,7 @@ catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
 
 # A valid Git branch from https://github.com/openshift/oadp-operator
-PREVIOUS_CHANNEL ?= oadp-1.2
+PREVIOUS_CHANNEL ?= oadp-1.3
```

Comment:
for master, should be 1.4 (highest release), but can be follow up PR :smile:

Reply:
@mateusoliveira43 Probably doesn't matter much here, but I'd say 1.4 would be appropriate once 1.4.0 is officially out.